src/index.html

```diff
@@ -11,22 +11,39 @@
       content="ie=edge"
     />
     <title>Document</title>
+    <link
+      rel="stylesheet"
+      href="fonts.css"
```

Comment:
Ensure that the path to `fonts.css` is correct relative to the location of this HTML file. If `fonts.css` is in the `src` directory, the path should be `./src/fonts.css`.